ChessClock/Clocks/ClocksPresenter.swift

```diff
@@ -28,18 +28,17 @@ class ClocksPresenter {
 
     init(view: ClockView) {
         let startState = ClocksViewModel(running: .paused, topTime: kStartingTime, bottomTime: kStartingTime)
+        
         self.view = view
-        Observable<ClocksViewModel?>.concat(
-            Observable.just(ClocksViewModel(running: RunningState.paused, topTime: kStartingTime, bottomTime: kStartingTime)),
-            Observable<Int>.interval(.seconds(1), scheduler: MainScheduler.instance)
-                .withLatestFrom(clocksStateBehaviourSubject.asObservable())
-                .map { event -> ClocksViewModel in
-                    let running = RunningState(event: event)
-                    return ClocksViewModel(running: running, topTime: kStartingTime, bottomTime: kStartingTime)
+        Observable<Int>.interval(.seconds(1), scheduler: MainScheduler.instance)
+            .withLatestFrom(clocksStateBehaviourSubject.asObservable())
+            .map { event -> ClocksViewModel in
+                let running = RunningState(event: event)
+                return ClocksViewModel(running: running, topTime: kStartingTime, bottomTime: kStartingTime)
             }
-        )
+            .startWith(startState)
```

Comment:
Nice one, by the time I started this project I didn't know about this operator, thank you 💪